src/components/PageTitle.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styles from '../styles/breadcrumb'
+import { translate } from '../lib/I18n'
+
+const PageTitle = ({pathname, t}) => {
+  let title
+  switch (pathname) {
+    case String(pathname.match(/^\/settings/)) :
+      title = 'mobile.settings'
```

Comment:
I never know if it is better to have in `locales/en.json`:
```json
{
  "settings": "Settings"
}
```

or

```json
{
  "settings": {
    "title": "Settings"
  }
}
```

But it may be easier to categorize by feature:
```json
{
  "settings": {
    "title": "Title",
    "version.number": "Version's number",
    "server.url": "Server's URL"
  }
}
```